README.md

```diff
@@ -145,16 +145,36 @@ There is only a limited number of such operations and they are only implemented
 ### Time Graph Tooltip
 Currently, the **Time Graph Tooltip** is populated when selecting a state in a Time Graph view.
 
+## New contributors
+⚠️ **Linux only!** The development environment for this project only works on Linux.
+
+### Explore the project context
+* **Check out the [tracevizlabs](https://github.com/tuxology/tracevizlab).** You don't need to complete all of them, but reading a couple is good context. Lab cycles 0 and 1 are especially useful.
+* **Browse recent conference presentations.** They give an overview of the trace viewer's goals, architecture, and functionality.
+  * _Using Theia to take trace analysis and visualization to the next level_, [slides](https://www.eclipsecon.org/sites/default/files/slides/EclipseConEurope2019-TraceCompass-Theia.pdf) - Bernd Hufmann, Ericsson AB (EclipseCon 2019)
+  * _A New Flexible Architecture for Trace Compass_, [slides](https://tracingsummit.org/ts/2019/files/Tracingsummit2019-theia-dagenais.pdf), [video](https://www.youtube.com/watch?v=8s5vGf45e-g) - Michel Dagenais, Polytechnique Montréal ([Tracing Summit 2019](https://tracingsummit.org/ts/2019/))
+* **Get an [overview of external components](https://github.com/theia-ide/theia-trace-extension#related-code)** used by this trace viewer. See how they interact.
+
+### Start with a small change
+* **Explore the [Gitpod demo](https://github.com/theia-ide/theia-trace-extension#try-a-live-demo-via-gitpod).** Experiment with the trace viewer. Start exploring its code via Gitpod's in-browser IDE.
+* **Start with a miniscule frontend improvement** like a [less than 5 line UI change](https://github.com/theia-ide/theia-trace-extension/pull/369/files). The Theia-based frontend repo is much less complex than the Trace Server's, so by starting in the frontend you'll learn high-level context about the project's different components without being overwhelmed with complexity.
```

Comment:
Maybe change "miniscule" to "small". Also, change "less than 5 line UI change" to something else. Maybe, "only a few lines to change in the UI".

Reply:
I'd like to give a concrete reference for how small the diff should be. Is less than 5 lines too restrictive? Could I say less than 10 lines?

Reply:
I went with, "**Start with a small frontend improvement.** A relevant change can be [less than 5 lines of code](https://github.com/theia-ide/theia-trace-extension/pull/369/files)."

Let me know what you think!